src/Ev.ServiceBus/Management/Factories/MessageSenderFactory.cs

```diff
@@ -30,7 +30,6 @@ public class MessageSenderFactory
     public IMessageSender CreateSender(ClientOptions[] senderOptions)
     {
         var options = (IClientOptions)senderOptions.First();
-        _logger.LogInformation("[Ev.ServiceBus] Initialization of sender client '{ResourceId}': Start", options.ResourceId);
```

Comment:
Please keep this log. 
it is only produced during initialization of the app, so the cost of having them is low.
And they are useful to know exactly which resource are initialized